### ISSUE TITLE
Simularor fails to compile and launch

### ISSUE THREAD

**Issue by Santara** [TianyuGuCMU/Simulator#2]
Hi,

I have solved all the dependencies of this simulator but the program fails to compile and launch. It crashes with the following error message:

__Starting /data/Anirban/Simulators/Simulator/build-dev-Desktop-Debug/dev...
The program has unexpectedly finished.
/data/Anirban/Simulators/Simulator/build-dev-Desktop-Debug/dev crashed__

I found this thread: https://forum.qt.io/topic/18827/solved-the-program-has-unexpectedly-finished/12 
which discusses this error message from qtcreator.

Please help me fix this up. I am extremely interested in using your simulator for research.

Thank you
Anirban

**Comment by mrasyid** [comment]
I have the same problem as well. Any solution will be wonderful.

**Comment by peterwludwig** [comment]
Santara, did you have any luck finding the issue?

**Comment by Santara** [comment]
@peterwludwig nope! I gave up hope and moved on to develop my own on top of gym-torcs: https://github.com/ugo-nama-kun/gym_torcs

**Comment by wuaaron5** [comment]
@Santara Hi, i try to use this simulator and still have the problem about dependencies. It can't find the lcsv_parser. What's this ? What should i install?
thx

**Comment by TianyuGuCMU** [comment]
Hey, thanks for your feedback. I have not been actively maintaining this repository... and this code is not very well structured as it appears to me now.

I have been working on a better simulation system (mostly for performing experiments for my PhD thesis). I am planning to make it open source. It is still QT-based, but makes use of google's tools like protobuf, gtest and bazel build system. 

Stay tuned.
